test/test_ipython_extension.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+from IPython.testing.globalipapp import start_ipython
```

Comment:
IPython will need to be added to `test/requirements.txt`.

Reply:
Thanks!